### PR TITLE
Crossdock echo behaviors

### PR DESCRIPTION
**Pull request by breerly**
- [x] implement 3 echo behaviors tests in async
- [x] implement 3 echo behaviors in sync
- [x] run against other langs `crossdock-echo-behaviors` branch

+~300 LOC without licenses

cc @yarpc/python

### CODE REVIEW

File: tests/yarpc/test_thrift.py [hunk 1]
```diff
@@ -3,15 +3,17 @@
 )
 
 import pytest
+import ipdb  # noqa
```
Comment (breerly) [inline tests/yarpc/test_thrift.py]: ugh

File: crossdock/async/client/behavior/raw.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+from tornado import gen
+
+from crossdock import rand, transport
+from yarpc import RPC, Request
+
```
Comment (kriskowal) [inline crossdock/async/client/behavior/raw.py]: double line

File: crossdock/__init__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
```
Comment (abhinav) [inline crossdock/__init__.py]: Empty files don't need license headers

File: crossdock/async/client/behavior/raw.py (hunk 2 above)
Comment (breerly) [inline crossdock/async/client/behavior/raw.py]: python wants it like dat :)

File: crossdock/async/client/behavior/__init__.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+from .json import json  # noqa
+from .raw import raw  # noqa
+from .thrift import thrift  # noqa
```
Comment (abhinav) [inline crossdock/async/client/behavior/__init__.py]: Prefer `__all__` over `# noqa`

Comment (breerly) [inline crossdock/async/client/behavior/__init__.py]: https://github.com/yarpc/yarpc-python/issues/44

File: crossdock/__init__.py (hunk 3 above)
Comment (breerly) [inline crossdock/__init__.py]: roger

**Comment by kriskowal** [comment]
:ok:

**Comment by abhinav** [comment]
:ok:

**Pull request merged**